spec/models/course/exp_record_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Course::ExpRecord, type: :model do
+  it { is_expected.to belong_to(:course_user).inverse_of(:exp_records) }
+  it { is_expected.to validate_presence_of(:course_user) }
+  it { is_expected.to validate_numericality_of(:exp_awarded).only_integer }
+
+  let!(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    context 'when manual' do
+      let(:manual_exp_record) { FactoryGirl.build :manual_exp_record }
+
+      it { expect(manual_exp_record).to be_valid }
+
+      it '.is_manual_exp? is true' do
+        expect(manual_exp_record.is_manual_exp?).to be_truthy
+      end
+
+      context 'has no reason' do
```

Comment:
By convention context should be followed by <code>with</code> or <code>when</code>
so
```
context 'when where is no reason'
```